body/stretch_body/stepper.py

```diff
@@ -564,7 +564,7 @@ def menu_transaction(self,x,do_print=True):
             while self.transport.ser.inWaiting():
                 r=self.transport.ser.readline()
                 if do_print:
-                    print r,
+                    print(r)
```

Comment:
This prints a new line, while the original did not. You can fix that by adding this import and changing the print command to:
```python
from __future__ import print_function
...
print(r, end=' ')
```